paddle/fluid/operators/conv_cudnn_op.cu.cc

```diff
@@ -158,6 +157,16 @@ class CUDNNConvOpKernel : public framework::OpKernel<T> {
       VLOG(5) << "NOT use cudnn_tensor_op_math";
     }
 #endif
+    Tensor cudnn_workspace;
+    void* cudnn_workspace_ptr = nullptr;
+    if (exhaustive_search && (!half_float)) {
```

Comment:
How about move this code to Line 179? Maybe codes would be more readable if moved.